Cubical/Categories/Displayed/Functor.agda

```diff
@@ -157,3 +172,56 @@ module _ {C : Category ℓC ℓC'} {D : Category ℓD ℓD'}
   ∫F .F-hom (_ , fᴰ) = _ , Fᴰ.F-homᴰ fᴰ
   ∫F .F-id = ΣPathP (_ , Fᴰ.F-idᴰ)
   ∫F .F-seq _ _ = ΣPathP (_ , (Fᴰ.F-seqᴰ _ _))
+
+-- Projections out of the total category
+module _ {C : Category ℓC ℓC'} {Cᴰ : Categoryᴰ C ℓCᴰ ℓCᴰ'} where
+  open Functor
+  open Functorᴰ
+
+  Fst : Functor (∫C Cᴰ) C
+  Fst .F-ob = fst
+  Fst .F-hom = fst
+  Fst .F-id = refl
+  Fst .F-seq =
+    λ f g → cong {x = f ⋆⟨ ∫C Cᴰ ⟩ g} fst refl
+
+  -- Functor into the total category
+  module TotalCategory {D : Category ℓD ℓD'}
```

Comment:
Should this be a named module? The analogs of this, e.g., `Data.Sum.elim` are not in named modules, you just tend to have to import the module `as Sum` to use them without name clashes.